src/content/changes/highlights/2023-08-31.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2023-08-31T14:00:00
+title: Highlights for the week ending August 31 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## aws-load-balancer
+- [aws-load-balancer-controller-app](https://github.com/giantswarm/aws-load-balancer-controller-app) version [v1.3.4](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#134---2023-08-28), migrates from monitoring labels to ServiceMonitor and introduces a new psp for CAPA support.
+## external-dns
+- [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.
```

Comment:
```suggestion

## external-dns
- [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.
```